packages/openapi-to-graphql/test/example_api.test.ts

```diff
@@ -2274,18 +2272,17 @@ test('Use headers option as function', () => {
     })
 })
 
-test('Use requestOptions headers option as function', () => {
+test('Use requestOptions option as function', () => {
   const options: Options<any, any, any> = {
-    requestOptions: {
-      headers: (method, path, title) => {
-        if (method === 'get' && path === '/snack') {
-          return {
+    requestOptions: (method, path, title) => {
```

Comment:
Example of using requestOptions as a function.

Reply:
Looks good